Automattic/wp-calypso
# Limit Global Styles: Add new customization features to plan comparison

Partially fixes https://github.com/Automattic/dotcom-forge/issues/899
Take two of https://github.com/Automattic/wp-calypso/pull/68265


#### Proposed Changes

Adds back the new customization features that were added by https://github.com/Automattic/wp-calypso/pull/68265 and later reverted by https://github.com/Automattic/wp-calypso/pull/68818.

Before | After
--- | ---
<img width="1070" alt="Screen Shot 2022-10-11 at 15 26 02" src="https://user-images.githubusercontent.com/1233880/195332784-6b9b60aa-6326-4f6f-a6cb-9e332661fd1a.png"> | <img width="1080" alt="Screen Shot 2022-10-11 at 15 27 44" src="https://user-images.githubusercontent.com/1233880/195332809-d6a15a6f-cb83-47df-b41c-9a7a20e7f3ad.png">

The problems that caused the revert are now addressed:
- The PR has the "String Freeze" label to block the merge until translations are complete 
- The feature constants have been cleaned up to better reflect the current state.
- We excluded the signup flow plans step from these changes since we're running a few experiments and tightly controlling the features that get listed there.

#### Testing Instructions

- Use the Calypso live link below
- Switch to a free site
- Go to Upgrade > Plans
- Make sure all the paid plans include "Advanced design customization" and "CSS Customization"
